app/src/main/java/pm/gnosis/heimdall/data/remote/RelayServiceApi.kt

```diff
@@ -14,24 +14,24 @@ interface RelayServiceApi {
     @POST("v1/safes/{address}/transactions/estimate/")
     fun estimate(@Path("address") address: String, @Body params: EstimateParams): Single<RelayEstimate>
 
-    @POST("v2/safes/")
-    fun safeCreation(@Body params: RelaySafeCreationParams): Single<RelaySafeCreation>
+    @GET("v1/tokens/?limit=1000&ordering=relevance,name&gas=true")
```

Comment:
you can as well change the limit to 3000 (not related to this pr, though) 😉

Reply:
yes, this is not part of this PR ;)